pool/web/view/component/component_notification.ml

```diff
@@ -7,6 +7,7 @@ let style_to_class = function
   | `Warning -> "warning"
 ;;
 
+(* TODO[timhub]: Why is the link content hardcoded? *)
```

Comment:
fix todo ;-)

- may use a `?(link : string * I18n.t)` tuple as an optional input argument
- use `CCOption.map_or ~default:(txt "") (fun (path, label) -> ... ) link`

Reply:
I guess that was the idea :)